daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -138,12 +138,19 @@ NAME     	CHART VERSION	APP VERSION	DESCRIPTION
 dapr/dapr	1.0.0-rc.1   	1.0.0-rc.1 	A Helm chart for Dapr on Kubernetes
 ```
 
-The APP VERSION column tells us which Dapr runtime version is installed by the chart.
+The APP VERSION column tells us which Dapr runtime version is installed by the chart. Now, use the following command to upgrade Dapr to your desired runtime version providing a path to the certificate files you saved before:
 
-Use the following command to upgrade Dapr to your desired runtime version providing a path to the certificate files you saved:
+> Remove the `--set global.ha.enabled=true` if your current Dapr installation has not been deployed is HA mode.
```

Comment:
in HA mode

Reply:
fixed

Reply:
Not seeing it come though quite yet, still reads `is HA mode`